app-crypt/efitools/efitools-1.8.1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic
+
+DESCRIPTION="Tools for manipulating UEFI secure boot platforms"
+HOMEPAGE="https://git.kernel.org/cgit/linux/kernel/git/jejb/efitools.git"
+SRC_URI="https://git.kernel.org/pub/scm/linux/kernel/git/jejb/efitools.git/snapshot/efitools-1.8.1.tar.gz"
+
+LICENSE="GPL-2 LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="libressl"
+
+RDEPEND="!libressl? ( dev-libs/openssl:0= )
+	libressl? ( dev-libs/libressl:0= )
+	sys-apps/util-linux"
+
+DEPEND="${RDEPEND}
+	app-crypt/sbsigntool
+	dev-perl/File-Slurp-Unicode
+	sys-apps/help2man
+	sys-boot/gnu-efi
+	virtual/pkgconfig"
+
+# S=${WORKDIR}/v${PV}
+
+src_prepare() {
+	# Respect users CFLAGS
+	sed -i -e '/CFLAGS.*=.*/d' Make.rules || die
```

Comment:
Nonono, don't reinvent the Makefile here. Just replace the `-O2` with `$(CFLAGS)` or `= -O2` with `+=`.

Reply:
(and maybe `-g` if it's not strictly necessary)

Reply:
FWICS you also need to fix `LDFLAGS`.

Reply:
Okay, CFLAGS with += and removing "-O2 -g" seems to do the job. Pushed a new one.
But I don't know, what to do about LDFLAGS. The problem is, the Make calls ```ld``` directly. So when I do trow my LDFLAGS in, this will fail e.g. with ```.ld: unknown option »-Wl,-O1« ```

Reply:
I've commited a try for fixing those LDFLAGS. Renamed the existing LDFLAGS to LIBS, and added $(LDFLAGS) to $(CC) lines. But that seems not to help, as QA still reports about LDFLAGS.

Reply:
Found a solution. Adding $(LDFLAGS) to Makefile and renaming LDFLAGS to LIBS in Make.rules makes the QA LDFLAGS warning go away. Commited.